### PR TITLE
feat: add migrations

### LINKED ISSUE #14: Implementacja migracji i modeli CZ 2
https://dbdiagram.io/d/6384d83fc9abfc611175acef

Image

### PR DESCRIPTION
**Pull request by szymon-kmiec**
Added part 2 migrations: Genders, Lists, People, Casts, Departments, Crews
https://github.com/skni-kod/FilmweebBack/issues/14

### CODE REVIEW

File: database/migrations/2022_12_01_195658_create_departments_table.php [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('departments', function (Blueprint $table) {
+            $table->id();
+            $table->unsignedBigInteger('user_id');
```
Comment (marioooo0o) [inline database/migrations/2022_12_01_195658_create_departments_table.php]: why???

File: database/migrations/2022_12_01_195658_create_departments_table.php [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+            $table->string('name',100);
+            $table->timestamps();
+
+            $table->foreign('user_id')->references('id')->on('users')->onDelete('cascade');
```
Comment (marioooo0o) [inline database/migrations/2022_12_01_195658_create_departments_table.php]: why??

File: app/Models/Medium_list.php [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+class Medium_list extends Model
+{
+    use HasFactory;
+
+    protected $fillable = ['user_id','name'];
+
+    public function user(){
+        return $this->belongsTo(User::class);
+    }
+
```
Comment (marioooo0o) [inline app/Models/Medium_list.php:17]: where relations genders??

File: app/Models/Crew.php [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class Crew extends Model
+{
+    use HasFactory;
+
+    protected $fillable=['medium_id','person_id','department_id','job'];
+    public function medium(){
```
Comment (marioooo0o) [inline app/Models/Crew.php:14]: one break line please 😄

File: app/Models/Cast.php [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+{
+    use HasFactory;
+
+    protected $fillable=['medium_id','person_id','gender_id','character','priority'];
+
+    public function medium(){
+        return $this->belongsTo(Medium::class);
+    }
+
+    public function people(){
```
Comment (marioooo0o) [inline app/Models/Cast.php]: person()

File: app/Models/Cast.php [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+        return $this->hasMany(Person::class);
```
Comment (marioooo0o) [inline app/Models/Cast.php]: belongsTo()

File: app/Models/Cast.php [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+    }
+
+    public function genders(){
```
Comment (marioooo0o) [inline app/Models/Cast.php]: gender()

File: app/Models/Cast.php [hunk 8]
```diff
@@ -0,0 +1,25 @@
...
+        return $this->hasMany(Gender::class);
```
Comment (marioooo0o) [inline app/Models/Cast.php]: belongsTo()

**Review by marioooo0o — CHANGES_REQUESTED** [review]
A few changes and it will be great! 😄

**Comment by szymon-kmiec** [comment]
Fixed

**Review by marioooo0o — APPROVED** [review]
I confirm

**Pull request merged**